tests/unit/test_network_connections.py

```diff
@@ -2503,10 +2504,13 @@ def test_path_to_glib_bytes(self):
 
 
 class TestUtils(unittest.TestCase):
-    def test_check_output(self):
-        res = Util.check_output(["echo", "test"])
-        self.assertEqual(res, "test\n")
-        self.assertRaises(n.MyError, Util.check_output, ["false"])
+    def test_decode_perm_addr(self):
+        if sys.version_info >= (3, 0):
+            mac_bytes = b"\xaa\xbb\xcc\xdd\xee\xff"
+        else:  # bytes is an alias for string type in python2
```

Comment:
I don't follow this comment. Can't you use b"..." in both python versions since this is what array.array.tostring() returns?

Reply:
You are right, ugh sorry for missing python2 stuff :-) Thanks!